oci-unit-tests/postgresql_test.sh

```diff
@@ -48,16 +60,28 @@ test_set_admin_user() {
     wait_postgres_container_ready "${container}" || return 1
     debug "Testing connection as ${admin_user}, looking for \"postgres\" DB"
     # default db is still "postgres"
-    out=$(psql postgresql://${admin_user}:${password}@127.0.0.1 -q -l -A -t -F % | grep "^postgres" | cut -d % -f 1)
+    out=$(docker run \
+              --network $DOCKER_NETWORK_NAME \
+              --rm \
+              ${image} \
```

Comment:
You call `docker run` like this multiple times, I'd create a function for it. IMHO It would increase the readability of the tests.

Reply:
I totally agree, but I'd like to do that in a separate PR.  IMO, the entire testcase can be improved by using more functions.